src/shared/layouts/BaseLayout/components/TopBar/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -10,21 +10,16 @@ import {
 } from 'shared/styles/styledComponents';
 import { variables } from 'shared/styles/variables';
 
-import {
-  StyledLink,
-  StyledBox,
-  StyledIconWrapper,
-  StyledIconImg,
-  StyledPlaceholder,
-} from './Breadcrumbs.styles';
+import { StyledLink, StyledBox, StyledIconWrapper, StyledPlaceholder } from './Breadcrumbs.styles';
+import { BREADCRUMB_ICON_SIZE } from './Breadcrumbs.const';
 
 export const Breadcrumbs = () => {
   const breadcrumbsData = breadcrumbs.useData();
 
-  const getBreadcrumbIcon = (icon: string | JSX.Element, label: string) => (
+  const getBreadcrumbIcon = (icon: string, label: string) => (
     <StyledIconWrapper>
       {icon ? (
-        <>{typeof icon === 'string' ? <StyledIconImg src={icon} alt="Icon" /> : icon}</>
+        <Svg id={icon} width={BREADCRUMB_ICON_SIZE} height={BREADCRUMB_ICON_SIZE} />
```

Comment:
If an applet has an image, which is received from the back-end than we need to show it instead of icon
pls take a look on screensots:
![image](https://user-images.githubusercontent.com/116339462/230564824-7566c03e-f4c1-4f90-894d-ec63fe74c818.png)
![image](https://user-images.githubusercontent.com/116339462/230564975-9c570b4d-b50d-4723-8f8b-93dd3c80e515.png)